packages/core/test/meta.test.ts

```diff
@@ -72,8 +72,9 @@ describe('state meta data', () => {
     }
   });
 
-  it('states should aggregate meta data', () => {
+  it.only('states should aggregate meta data', () => {
```

Comment:
this also is most likely not an intentional change 😉

Reply:
We use [`eslint-plugin-jest`](https://www.npmjs.com/package/eslint-plugin-jest) to catch these in the linter and fail builds if any test is focused using the [`no-focused-tests`](https://github.com/jest-community/eslint-plugin-jest/blob/HEAD/docs/rules/no-focused-tests.md) rule. Highly recommended, it's saved me from myself many times.